docs/use-github-in-bcgov/github-enterprise-user-licenses-bc-government.md

```diff
@@ -32,52 +34,52 @@ For more information on our GitHub organizations and their uses, see [B.C. gover
 
-Each ministry has GitHub administrators that manage user licences purchased by the ministry and assign individual users to the licences. All license management and licence-to-user assignment must go through ministry account administrators.   
+Once you have received a response from CSAM with a quote number and the cost, you or someone in your business area will need to  use this to create an iStore order referencing the quote number to initiate the purchase. The usual e-approval steps will take place, and CSAM will notify you by email once they have made the purchase on your behalf. 
 
-1. Your ministry’s GitHub administrator creates a private organization for your ministry, if it doesn’t exist already, and links it to the GitHub Enterprise account.
+Once CSAM has purchased licenses on your behalf, billing back to your expense authority will commence. Billing occurs monthly until you notify CSAM to cancel the user licenses.
 
-2. Submit a request to Software Central Management to purchase the desired number of user licences.
+## Getting access to GitHub Enterprise
 
-3. Once the order is processed and the user licenses have been added to the Province’s GitHub Enterprise account, your ministry’s GitHub administrator adds your users to the GitHub Enterprise account and to the ministry’s private organization.
+Once licenses have been acquired, you'll need to request access through your ministry or sector IMB to the corresponding organization within B.C. government's GitHub Enterprise environment. 
 
-These users have access to all features available in GitHub Enterprise within their private organization.
+Once you've been granted access, you'll be able to log in to the B.C. government GitHub Enterprise at [https://github.com/enterprises/bcgov-ent/](https://github.com/enterprises/bcgov-ent/) using your IDIR and credentials begin working in private repositories within your ministry or sector's organization. 
 
-## Pay for the GitHub Enterprise user licences
+## Background: BC government GitHub Enterprise process map
 
-Software Central Management handles your purchase order, with billing back to your expense authority. The ministry is billed monthly for the GitHub Enterprise user licenses until Software Central gets a request asking to cancel the user licenses.
+Our team has created the artifact below which captures the steps and teams involved in acquiring and mananaging GitHub Enterprise licenses. This version is intended for reference only, but we intend to create simplified, more focused versions for both end users and administrators.  
 
+![A detailed diagram illustrating the process of acquiring and administering GitHub Enterprise licenses within the B.C. government. ](../images/github-e-process-map.png)
```

Comment:
Hi @SHIHO-I !

Congratulations on your first pull request!

I fixed the link, moved the location of the diagram and added "alt text", which is important for accessibility.